build/package/Microsoft.DotNet.Cli.Installer.DEB.targets

```diff
@@ -86,14 +93,8 @@
       <DebianConfigTokenValues Include="%SHARED_FRAMEWORK_DEBIAN_PACKAGE_NAME%">
         <ReplacementString>$(SharedFxDebianPackageName)</ReplacementString>
       </DebianConfigTokenValues>
-      <DebianConfigTokenValues Include="%SHARED_FRAMEWORK_NUGET_NAME%">
```

Comment:
No, the CLI repo doesn't produce the real Core-Setup DEB packages.  The CLI only produces a single DEB package: `dotnet-dev-x.y.z`.  It consumes the Core-Setup DEB packages though by depending on them, and downloads them so they can be installed during the test.

These deleted token values were just left over, and I couldn't find any usages of them so I deleted them.